msgpack-core/src/main/java/org/msgpack/core/MessagePack.java

```diff
@@ -267,7 +276,8 @@ public boolean equals(Object obj)
             PackerConfig o = (PackerConfig) obj;
             return this.smallStringOptimizationThreshold == o.smallStringOptimizationThreshold
                     && this.bufferFlushThreshold == o.bufferFlushThreshold
-                    && this.bufferSize == o.bufferSize;
+                    && this.bufferSize == o.bufferSize
```

Comment:
@xerial We'd better override `hashCode` when overriding `equals`. Let's do that later